lib/routes/scut/gzic/media.ts

```diff
@@ -0,0 +1,71 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import ofetch from '@/utils/ofetch';
+
+export const route: Route = {
+    path: '/gzic/media',
+    categories: ['university'],
+    example: '/scut/gzic/media',
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: true,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '广州国际校区 - 媒体报道',
+    maintainers: ['gdzhht'],
+    handler,
+    description: `:::warning
+    由于学校网站对非大陆 IP 的访问存在限制，可能需自行部署。
+    :::`,
```

Comment:
```suggestion
    description: `:::warning
由于学校网站对非大陆 IP 的访问存在限制，可能需自行部署。
:::`,
```
Indent description with 0 or 2 spaces.